src/api/routers/participantsRouter.ts

```diff
@@ -94,6 +98,20 @@ export function createParticipantsRouter() {
     }
   );
 
+  participantsRouter.get('/approved', isApproverCheck, async (req, res) => {
+    const participants = await getParticipantsApproved();
+    const sites = await getSiteList();
+    const allParticipantTypes = await ParticipantType.query();
```

Comment:
We are basically doing the same thing here as we are in https://github.com/IABTechLab/uid2-self-serve-portal/blob/0db97a0ae8be4bcaf987ef257838fecdc8169af0/src/api/routers/usersRouter.ts#L59. Just that this is for all approved participants and the `usersRouter` is just for the current participant. Perhaps this warrants extracting to a separate method?

Reply:
true, I guess it's sufficiently different now.